containerd-shim/kata/service.go

```diff
@@ -667,3 +650,24 @@ func (s *service) getContainer(id string) (*Container, error) {
 
 	return c, nil
 }
+
+func (s *service) getContainerStatus(containerID string) (task.Status, error) {
+	cStatus, err := s.sandbox.StatusContainer(containerID)
+	if err != nil {
+		return 0, err
```

Comment:
here if get the container status failed, it's better to return task.StatusUnknown instead of "0"

Reply:
ok. I will replace it.